src/shadowbox/server/manager_metrics.ts

```diff
@@ -40,4 +58,21 @@ export class PrometheusManagerMetrics implements ManagerMetrics {
     }
     return {bytesTransferredByUserId: usage};
   }
+
+  async getTunnelTimeByLocation({
```

Comment:
This is no longer by location. It's broken down by all dimensions. Please rename.